### PR TITLE
[Requirements] Add 'notebook' upper bound

### PR DESCRIPTION
**Pull request by rokatyy**
Jira - https://jira.iguazeng.com/browse/NUC-73

`notebookapp` package was fully deleted since `7.0.0`

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 nbconvert = ">=6.4.5"
-notebook = ">=6.4"
+notebook = ">=6.4, <7.0.0"
```
[the remark this answers is not in the document]
Comment (rokatyy) [reply Pipfile:8]: @jond01 1 - yes, I'm going to create another ticket to bump the `notebook` version to 7+; 2 - sure :)

Comment (alonmr) [inline Pipfile:8]: please add a comment explaining what has changed. why is this needed?

Comment (rokatyy) [reply Pipfile:9]: added
